src/lib/gui/core/ServerConnection.cpp

```diff
@@ -93,7 +93,8 @@ void ServerConnection::handleNewClient(const QString &clientName)
   Q_EMIT messageShowing();
 
   m_messageShowing = true;
-  const auto result = m_pDeps->showNewClientPrompt(m_pParent, clientName);
+  const auto result =
+      m_pDeps->showNewClientPrompt(m_pParent, clientName, m_appConfig.tlsEnabled() && m_appConfig.requireClientCerts());
```

Comment:
Does it make sense to call the 3rd bool arg `previouslyAccepted`? 
Wouldn't something like `requireClientCerts` make sense?

```suggestion
  const auto result =
      m_pDeps->showNewClientPrompt(m_pParent, clientName, m_appConfig.tlsEnabled() && m_appConfig.requireClientCerts());
```

Reply:
Nope when we call this if we have peer auth or encryption then the use has all ready accepted that they want the client to be able to connect.  so `previouslyAccepted` is what that represents.

Reply:
I don't see how this makes sense. How can `requireClientCerts` returning true prove that a client has been previously accepted? All it proves is that you checked the 'Require client certificates' checkbox.